src/corpus/corpus.cpp

```diff
@@ -3,9 +3,9 @@
  * @author Sean Massung
  */
 
+#include "meta/corpus/corpus.h"
 #include "cpptoml.h"
 #include "meta/corpus/all.h"
-#include "meta/corpus/corpus.h"
```

Comment:
Hm. Ok. [This](http://llvm.org/docs/CodingStandards.html#include-style) makes a convincing argument, so I guess we should just leave it.